src/workloads/query/BooleanSimplifier.yml

```diff
@@ -17,11 +17,7 @@ AutoRun:
         $eq:
           - replica
           - replica-all-feature-flags
-          - standalone
-          - standalone-80-feature-flags
-          - standalone-all-feature-flags
-          - standalone-classic-query-engine
-          - standalone-heuristic-bonsai
-          - standalone-sampling-bonsai
+          - replica-80-feature-flags
```

Comment:
Again, I think `replica-80-feature-flags` is redundant. Also, (I don't have context on this feature) do we want to test this with SBE?

Reply:
There was no SBE variant for this test originally, so I'll probably leave it as-is.